beta/aws-ecsfargate-cfn/README.md

```diff
@@ -0,0 +1,124 @@
+
+## In this folder
+
+- [`README.md`](./README.md): the document that you are reading. 👋😃
+- [`op-scim-bridge.yaml`](./op-scim-bridge.yaml): a [CloudFormation template](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-whatis-concepts.html#cfn-concepts-templates) for 1Password SCIM bridge.
+
+## Overview
+
+The included template can be used to deploy 1Password SCIM bridge as a [stack](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-whatis-concepts.html#cfn-concepts-stacks) using the [AWS CloudFormation console](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-using-console.html) or [the AWS Command Line Interface (AWS CLI)](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-using-cli.html).
+
+The CloudFormation template will create and configure the following resources in your AWS account:
+
+- a VPC (with a `10.0.0.0/16` CIDR block by default)
+- 2 public subnets (each with `/20` CIDR blocks within this range), spanned across two availability zones in the current or selected region
+- an internet gateway
+- an Application Load Balancer (ALB)
+- an ECS task with associated container definitions for 1Password SCIM bridge and the requred Redis cache
+- an ECS service and ECS cluster for the above task
+- a target group targeting the SCIM bridge container
+- 2 security groups to restrict traffic to and from the ALB and the ECS cluster, respectively
+- an AWS Certificate Manager (ACM) TLS certificate for the ALB
+- 2 Route 53 DNS `A` records:
+  - for the domain name of your SCIM bridge (e.g. `scim.example.com`)
+  - to validate the requested ACM certificate
+- an AWS Secrets Manager secret to store the `scimsession` credentials for your SCIM bridge
+- a CloudWatch log group to capture logs in streams from each of the SCIM bridge and Redis containers
+- required IAM roles
+
+This template is a working example to be used as a base for your SCIM bridge deployment and may need to be customized to meet your specific requirements.
+
+## Prerequisites
+
+- A 1Password account with an active 1Password Business subscription or trial
+  > **Note**
+  >
+  > 👀 Try 1Password Business **free** for 14 days: <https://start.1password.com/sign-up/business>
+- An AWS account with the permissions and available quota to create and manage the described resources
+- [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) (if you want to manage the deployment using a terminal or script).
+
+## Getting started
+
+Before deploying 1Password SCIM bridge, consult the [Preparation Guide](/PREPARATION.md) in this repository. Since this CloudFormation template will create all necessary resources, you do not need to seperately create a DNS record for your SCIM bridge, nor use its certificate manager component to create and manage a TLS certificate.
```

Comment:
1Password SCIM Bridge

Reply:
The `b` is supposed to be lower case in this product name (or at least *was*).